src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
-public class ColorSupplier {
+import java.util.Random;
+
+public class ColorSupplier extends Ball {
     public String getRandomColor() {
-        return null;
+        int random = new Random().nextInt(BallCollor.values().length);
+        return BallCollor.values()[random].toString().toLowerCase();
```

Comment:
Instead of using toString() and converting it to lowercase, you should use the name() method to get the string representation of the enum constants as per the checklist.